src/ui/HeroBackground.jsx

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 
 const HeroBackground = styled.div`
-  height: 80dvh;
+  height: 80svh;
```

Comment:
Fixed rendering of the header on phone devices, the header now goes down animatedly smoothly, no more cutting and sudden movement